### PR TITLE
[stable/cerebro] Add ability to set tls skip verify via vaules file

### PR DESCRIPTION
**Pull request by rgarcia89**
#### What this PR does / why we need it:
In order to use elastic search via ssl with self signed certificates

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Review by rgarcia89 — COMMENTED** [review]
fixed true false behavior

File: stable/cerebro/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: cerebro
-version: 1.4.0
+version: 1.4.1
```
Comment (davidkarlsen) [inline stable/cerebro/Chart.yaml]:
```suggestion
version: 1.5.0
```

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Pull request merged**